app/src/main/java/com/example/coronadiagnosticapp/ui/fragments/instruction/InstructionViewModel.kt

```diff
@@ -5,5 +5,4 @@ import com.example.coronadiagnosticapp.data.repository.Repository
 import javax.inject.Inject
 
 class InstructionViewModel @Inject constructor(val repository: Repository) : ViewModel() {
-    fun getUserName() = repository.getUserName()
```

Comment:
Does this file have another purpose? Should it be deleted?

Reply:
I don't see a reason why not